pom.xml

```diff
@@ -121,7 +121,6 @@
             </plugin>
             <plugin>
                 <artifactId>maven-source-plugin</artifactId>
-                <version>3.0.0</version>
```

Comment:
I think we’re missing a version for the `maven-source-plugin` in `pluginManagement`.